pulp_container/app/__init__.py

```diff
@@ -7,3 +7,4 @@ class PulpContainerPluginAppConfig(PulpPluginAppConfig):
     name = "pulp_container.app"
     label = "container"
     version = "2.13.0.dev"
+    python_package_name = "pulp_container"
```

Comment:
Is this correct? Are we suppose to use the underscore or the dash version of the name. In `setup.py` the name has a dash.

Reply:
Dang this is the one I didn't look up. It is with the `-` https://pypi.org/project/pulp-container/